### ISSUE TITLE
[Remoto] Software Engineering Manager I Back end Kotlin - Loggi

### ISSUE THREAD

**Issue by Franciellidns** [backend-br/vagas#11270]
## Nossa empresa

Somos uma empresa de tecnologia que está reinventando a logística e a forma de fazer entregas no Brasil.

Queremos empoderar pessoas e transformar negócios com entregas de excelência. Movimentamos mais de 400 mil pacotes por dia com uma tecnologia de ponta a ponta, que aproxima todos os cantos do país.

Temos o propósito de que as entregas sejam acessíveis para todas as pessoas do Brasil. Esse é nosso desafio diário, por isso buscamos inovação em tudo que fazemos. 

Por aqui, valorizamos diferentes formas de pensar, pois sabemos que só assim poderemos criar um futuro mais democrático para as pessoas.

Nossa cultura é traduzida todos os dias pelos comportamentos de cada Logger e das experiências que construímos com os nossos clientes e sociedade.
Nossos valores são como um guia de como as pessoas devem se comportar diante das pequenas e grandes atitudes que são tomadas no nosso ambiente de trabalho todos os dias.

## Descrição da vaga

As equipes de Engenharia da Loggi são apaixonadas por desafios e buscam escalar um negócio transformador em um ambiente de constante aprendizado, tendo como compromisso a excelência para criar sistemas e otimizar processos, construindo assim uma plataforma de logística eficiente e inovadora. Nossos desafios são de alta complexidade, mesclando profissionais experientes e em início de carreira, em um ambiente fortalecido pelo aprendizado e compartilhamento de experiências.

Usamos tecnologia para resolver problemas de eficiência e operacionalização, além de produzirmos uma solução de coletas e entregas dinâmica e única.  Temos também desafios em maximizar o sucesso de entrega e construir mecanismos de pagamento justos aos nossos entregadores.

Atualmente, usamos Python/Django, Kotlin/Micronaut; modelos de machine learning treinados com SKLearn e Keras/Tensorflow/Jax; aplicativos móveis híbridos e PWAs; algoritmos de otimização e nossas próprias soluções proprietárias. Também contamos com implantações de Istio/K8s no Azure e AWS. Para dados em geral, contamos com Looker, ELK e Grafana. 

Junte-se a nós e faça parte da equipe que está democratizando a logística e conectando o Brasil, proporcionando uma experiência de entrega excepcional aos clientes.

## Local

Remoto

## Requisitos

- Mais de 6 anos de experiência relevante em desenvolvimento de software e experiência comprovada com gestão de equipes. 
- Sólida experiência em Kotlin | Java
- Sólida experiência em micronaut
- Experiência com AWS
- Experiência com Kafka
- Experiência com Banco de dados mongo db, postgresql
- Conhecimentos em Camel
- Sólidos conhecimentos na construção, administração e solução de problemas de sistemas complexos e de grande porte;
- Experiência com integrações (APIs públicas, arquivo) 
- Capacidade de identificar problemas e sugerir melhorias de infraestrutura e/ou código;
- Diferencial atuação com python, fast api ou django

## Benefícios

- Vale refeição no cartão Flash;
- Vale Transfer no cartão Flash (apenas para trabalho presencial);
- Auxílio Home Office;
- Assistência Médica - Mensalidade custeada pela Loggi, com coparticipação em exames simples e consultas, inclusive para filhos de até 21 anos (se universitários, até 24);
- Plano Odontológico - 100% custeado pela Loggi, inclusive para filhos de até 21 anos (se universitários, até 24);
- Seguro de Vida pela SulAmérica Seguros;
- Auxílio creche para filhos com até 5 anos;
- Incentivo para desenvolvimento técnico anual;
- Gympass para desconto em academias;
- Parcerias Loggi: Zenclub, Edupass, Funcional Card;
- Loggi 4 free - 4 entregas grátis por mês;
- Empréstimo consignado (elegível após 3 meses de casa);
- Licença maternidade/paternidade de 6 meses para mães e 20 dias para pais;
- Elegibilidade para remuneração variável de curto e longo prazo.

## Contratação

CLT

## Como se candidatar

Página de carreiras da Loggi https://www.loggi.com/carreiras/vagas/
Linkedin: https://www.linkedin.com/jobs/view/3771838191

### Labels

Kotlin
Java
Micronaut
AWS
Mongo DB
Postgresql
Python
Django
Fast API

### Nível

- Gerente

### Regime

- CLT

### Alocação

- Remoto
